doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+
+Let $prec(A) = E_{\mathcal{D}(z|A)}[1_{f(x)=f(z)}]$ be the precision of an anchor. Note that the precision of an anchor is considered with respect to the set of points in the data distribution to which the anchor applies, $\mathcal{D}(z|A)$. We can consider the **coverage** of an anchor as the probability that $A(z)=1$ for any instance $z$ in the data distribution. The coverage tells us the proportion of the distribution to which the anchor applies. The aim here is to find the anchor that applies to the most extensive set of instances while satisfying $prec(A) \geq \tau$.
+
+__TODO__:
+- Include picture explanation of anchors
+
+Alibi finds anchors by building them up from the bottom up. We start with an empty anchor $A=\{\}$ and then consider the set of possible feature values from the instance of interest we can add. $\mathcal{A} = \{A \wedge a_0, A \wedge a_1, A \wedge a_2, ..., A \wedge a_n\}$. For instance, in the case of textual data, the $a_i$ might be words from the instance sentence. In the case of image data, we partition the image into super-pixels which we choose to be the $a_i$. At each stage, we will look at the set of possible next anchors that can be made by adding in a feature $a_i$ from the instance. We then compute the precision of each of the resulting anchors and choose the best. We iteratively build the anchor up like this until the required precision is met.
+
+As we construct the new set of anchors from the last, we need to compute the precisions of the next group to know which one to choose. We can't calculate this directly; instead, we sample from $\mathcal{D}(z|A)$ to obtain an estimate. To do this, we use several different methods for different data types. For instance, in the case of textual data, we want to generate sentences with the anchor words and ensure that they make sense within the data distribution. One option is to replace the missing words (those not in the anchor) from the instance with words with the same POS tag. This means the resulting sample will make semantic sense rather than being a random set of words. Other methods are available. In the case of images, we sample an image from the data set and then superimpose the super-pixels on top of it.
+
+**Pros**
+- Easy to explain as rules are simple to interpret
+- Is a black-box method as we just need to predict the value of an instance and don't need to access model internals
+- Can be parallelized and made to be much more efficient as a result
+- Although they apply to a local instance, the notion of coverage also gives a level of global insight
+
+**Cons**
+- This algorithm is much slower for high dimensional feature spaces
+- If choosing an anchor close to a decision boundary, the method may require a considerable number of samples from $\mathcal{D}(z|A)$ and $\mathcal{D}(z|A')$ to distinguish two anchors $A$ and $A'$.
```

Comment:
Good point, but would remove the maths and keep the discussion informal. Additionally, not just a large number of samples, but the resulting anchors will be long --> less interpretable.